autosubmit/autosubmit.py

```diff
@@ -1470,6 +1470,9 @@ def delete(expid: str, force: bool) -> bool:
         :raises AutosubmitCritical: If the experiment does not exist or if there are insufficient permissions.
         """
         experiment_path = Path(f"{BasicConfig.LOCAL_ROOT_DIR}/{expid}")
```

Comment:
This can be moved after your check, to avoid declaring it altogether in case the `check_directory_in_use` fails. :grimacing: